bdk-ffi/src/keys.rs

```diff
@@ -158,21 +184,27 @@ impl DescriptorSecretKey {
         secret_bytes
     }
 
+    /// Convert this secret key to a string.
     pub fn as_string(&self) -> String {
         self.0.to_string()
     }
```

Comment:
This is a vestige of the times where we could not get the Display trait on types.

I think we should add the Display in this case so the `.toString()` method is implemented for `DescriptorSecretKey`.